exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -97,32 +113,43 @@ class JaegerSpanExporter(SpanExporter):
             when query for spans.
         agent_host_name: The host name of the Jaeger-Agent.
         agent_port: The port of the Jaeger-Agent.
-        collector_endpoint: The endpoint of the Jaeger-Collector HTTP/HTTPS Thrift.
+        collector_endpoint: The endpoint of the Jaeger-Collector
+            Thrift over HTTP/HTTPS or Protobuf via gRPC.
         username: The user name of the Basic Auth if authentication is
             required.
         password: The password of the Basic Auth if authentication is
             required.
+        insecure: Connection type.
```

Comment:
If this is a boolean value, a more specific documentation would be something like `True if an insecure connection to ... is to be used`